e2e/process-services-cloud/process-custom-filters.e2e.ts

```diff
@@ -25,196 +25,381 @@ import { TasksCloudDemoPage } from '../pages/adf/demo-shell/process-services/tas
-            browser.driver.sleep(1000);
-            processCloudDemoPage.getAllRowsByIdColumn().then(function (list) {
-                const initialList = list.slice(0);
-                list.sort(function (firstStr, secondStr) {
-                    return firstStr.localeCompare(secondStr);
-                });
-                expect(JSON.stringify(initialList) === JSON.stringify(list)).toEqual(true);
+    it('[C305054] Should display processes ordered by status when Status is selected from sort dropdown', async () => {
+        processCloudDemoPage.editProcessFilterCloudComponent().clickCustomiseFilterHeader().setStatusFilterDropDown('ALL')
+            .setSortFilterDropDown('Status').setOrderFilterDropDown('ASC');
+        processCloudDemoPage.processListCloudComponent().getDataTable().checkSpinnerIsDisplayed().checkSpinnerIsNotDisplayed();
+        processCloudDemoPage.processListCloudComponent().getAllRowsStatusColumn().then(function (list) {
+            const initialList = list.slice(0);
+            list.sort(function (firstStr, secondStr) {
+                return firstStr.localeCompare(secondStr);
             });
+            expect(JSON.stringify(initialList) === JSON.stringify(list)).toEqual(true);
+        });
 
-            processCloudDemoPage.editProcessFilterCloudComponent().setOrderFilterDropDown('DESC');
-            processCloudDemoPage.processListCloudComponent().getDataTable();
-            browser.driver.sleep(1000);
-            processCloudDemoPage.getAllRowsByIdColumn().then(function (list) {
-                const initialList = list.slice(0);
-                list.sort(function (firstStr, secondStr) {
-                    return firstStr.localeCompare(secondStr);
-                });
-                list.reverse();
-                expect(JSON.stringify(initialList) === JSON.stringify(list)).toEqual(true);
+        processCloudDemoPage.editProcessFilterCloudComponent().setOrderFilterDropDown('DESC');
+        processCloudDemoPage.processListCloudComponent().getDataTable().checkSpinnerIsDisplayed().checkSpinnerIsNotDisplayed();
+        processCloudDemoPage.processListCloudComponent().getAllRowsStatusColumn().then(function (list) {
+            const initialList = list.slice(0);
+            list.sort(function (firstStr, secondStr) {
+                return firstStr.localeCompare(secondStr);
             });
+            list.reverse();
+            expect(JSON.stringify(initialList) === JSON.stringify(list)).toEqual(true);
         });
+    });
 
-        it('[C297697] The value of the filter should be preserved when saving it', async () => {
-            processCloudDemoPage.editProcessFilterCloudComponent().clickCustomiseFilterHeader()
-                .setProcessInstanceId(completedProcess.entry.id);
+    // bug raised for this ACTIVITI-3046
+    xit('[C305054] Should display processes ordered by start date when StartDate is selected from sort dropdown', async () => {
```

Comment:
Can we enable this test, remove the comment and add 'Waiting for dev fix' tag ?

Reply:
I would prefer to remove just this test and get the rest of the tests merged. and link a automation ticket to this bug.